src/Components/LayoutComponents/DrawerGroup.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import propTypes from 'prop-types';
```

Comment:
So this file, formstepgroup.js pagetitle.js and titlegroup.js are all the same level NDA file, applying Ah few classes to ah div, any chance we can make a single helper components that is passed the appropriate classes to accomplish this with far less code?

Reply:
Oh yeah, that makes sense!